scholia/config.py

```diff
@@ -0,0 +1,46 @@
+"""Configuration for Scholia."""
+from __future__ import absolute_import
+
+try:
+    import ConfigParser as configparser
+except ImportError:
+    import configparser
+
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
+
+import logging
+
+from os.path import exists, expanduser
+
+
+CONFIG_FILENAMES = [
+    'scholia.cfg',
+    '~/etc/scholia.cfg',
+    '~/scholia.cfg']
+
+DEFAULTS = """
+[servers]
+SPARQLEndPointURLbd = https://query.wikidata.org/bigdata/namespace/wdq/sparql
+SPARQLEndPointURL = https://query.wikidata.org/sparql
+SPAREndPointEmbed = https://query.wikidata.org/
```

Comment:
Shouldn't this be SPARQLEndpointEmbed or QueryServiceURL

Reply:
I have pushed a new batch of improvements. The naming wasn't consistent.